docs/tutorials/tutorial-custom-domain-with-istio.md

```diff
@@ -148,7 +148,7 @@ spec:
   cloudProfileName: gcp
   kubernetes:
     allowPrivilegedContainers: true
```

Comment:
You could remove this field. It is removed from the API: https://github.com/gardener/gardener/blob/d839b336dfd7576f6c4aff67dc0981c4e20209b2/pkg/apis/core/v1beta1/types_shoot.go#L586-L587

```suggestion
```